Sources/Public/Animation/LottieAnimation.swift

```diff
@@ -177,3 +177,7 @@ public final class LottieAnimation: Codable, DictionaryInitializable {
     })
   }()
 }
+
+// MARK: Sendable
+
+extension LottieAnimation: @unchecked Sendable { }
```

Comment:
Should we add a comment about why this is `@unchecked`?

Reply:
I switched this to use `Sendable` instead of `@unchecked Sendable`. This required marking all of the other data model types as `Sendable` as well, but that's fine since they're all immutable.